lib/src/core/room.dart

```diff
@@ -152,10 +150,10 @@ class Room extends DisposableChangeNotifier with EventsEmittable<RoomEvent> {
     String url,
     String token, {
     ConnectOptions? connectOptions,
-    RoomOptions? roomOptions,
```

Comment:
let's leave this here.. otherwise it would break folks apps. feel free to mark it as deprecated if that's the intention